mobile/src/profile/styles.ts

```diff
@@ -57,16 +60,16 @@ export const layoutStyles = StyleSheet.create({
     content: {
         flex: 1,
         flexDirection: 'column',
-        marginTop: placeholderHeight
-
+        marginTop: circleDiameter * 0.5 - headerGap,
+        height: 350
```

Comment:
can we use flex instead of hardcode?

Reply:
No, cause flex  does not work good for scrollView on Android.